src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs

```diff
@@ -58,12 +58,10 @@ internal sealed class UseExpressionBodyCodeRefactoringProvider() : SyntaxEditorB
     public override async Task ComputeRefactoringsAsync(CodeRefactoringContext context)
     {
         var (document, textSpan, cancellationToken) = context;
-        if (textSpan.Length > 0)
-            return;
 
         var position = textSpan.Start;
         var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-        var node = root.FindToken(position).Parent!;
+        var node = root.FindNode(textSpan);
```

Comment:
Can we now use the refactoring helper in this code (the code that locates a node of a particular type in th ecurrent context)?

Reply:
I've verified that this code path is utilized for both body->expression and expression->body conversions (required setting csharp_style_expression_bodied_methods to true/false to see both ways come through). I think I'd prefer not to use TryGetRelevantNodeAsync is it isn't completely clear to me which type to be using in both cases, and throwing in the lambda expression check right below it.

Reply:
@sharwell -- the current change makes sense to me and fits in well with how this method worked before. I'm willing to give the helper a try if you think it fits well here, just would like some clarification on which type(s) would be used on the call(s).

Reply:
@sharwell -- really want to move forward on this conversation.